clients/imodelhub/src/imodelbank/IModelBankHandler.ts

```diff
@@ -32,18 +31,16 @@ export class IModelBankHandler extends IModelBaseHandler {
 
   protected getUrlSearchKey(): string { assert(false, "Bentley cloud-specific method should be factored out of WsgClient base class"); return ""; }
 
-  public get baseUrl(): string {

```

Comment:
Wouldn't it be better to change the base class protected member to have leading underscore, rather than removing this getter? This exposes assignment outside of constructor.